NetKAN/StockLS.netkan

```diff
@@ -0,0 +1,34 @@
+{
+    "spec_version"   : "v1.4",
+    "name"           : "StockLS",
+    "abstract"       : "Life support applied to stock parts so ksp updates don't break functionality",
+    "identifier"     : "StockLS",
+    "download"       : "https://github.com/vosechu/StockLS/archive/v1.0.0.zip",
```

Comment:
You need to remove this line (ln 6) or else every time there's a new release CKAN will generate a new set of metadata but the download will still be this exact file. (ex. you release 1.1.0 but what users get installed is still 1.0.0)